availability.md

```diff
@@ -5,11 +5,11 @@ title: Magento Commerce product availability
 |-------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------|
+| **{{site.data.var.ee}} 2.3.1 GA**               | Available now                                                                                                                                                   | [Composer](https://devdocs.magento.com/guides/v2.3/install-gde/composer.html)                                                |
 | **{{site.data.var.ee}} 2.3.0 GA**               | Available now                                                                                                                                                          | [Composer](https://devdocs.magento.com/guides/v2.3/install-gde/composer.html)                                                |
-| **{{site.data.var.ee}} 2.3.1 GA**               | Available March 2019                                                                                                                                                   | [Composer](https://devdocs.magento.com/guides/v2.3/install-gde/composer.html)                                                |
 | **{{site.data.var.ece}} Tools (aka ECE-Tools)** | Available now                                                                                                                                                          | [Composer](https://devdocs.magento.com/guides/v2.3/cloud/project/ece-tools-update.html)                                      |
 | **PWA Studio**                                  | Available now including support for {{site.data.var.ece}}                                                                                                              | [Documentation](https://magento-research.github.io/pwa-studio/) and [GitHub](https://github.com/magento-research/pwa-studio) |
 | **Magento Shipping**                            | Available now                                                                                                                                                          | Merchants on Magento 2.2.2+ can use the [onboarding process](https://account.magento.com/shipping/onboarding/start)          |
-| **Page Builder**                                | Early adoption program is now closed. GA expected in March 2019 with 2.3.1 GA                                                                                          | Bundled with Magento Commerce 2.3.x                                                                                          |
+| **Page Builder**                                | Available now                                                                                          |
```

Comment:
I would suggest this addition directly, but the md table format confuses me... Will you please add the following line to the "How to get it" column for Page Builder?

> Bundled with Magento Commerce 2.3.x

Reply:
Yes, sorry. Not sure how I removed that. 🙍